### PR TITLE
Fix: Social Share - Fixed jQuery undefined issue.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello - https://trello.com/c/VKry98uR/281-fix-social-share-fixed-jquery-undefind-issue

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate social share block and drag and drop on pages.
- Check out the console log for error
- Drag and drop TOC, Inline notice, Forms, Blockquote, Lottie and check everything is working fine on the frontend. No issue in the console.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-helper.php [hunk 1]
```diff
@@ -187,7 +187,7 @@ public function generate_asset_files() {
 			global $content_width;
 			self::$stylesheet = str_replace( '#CONTENT_WIDTH#', $content_width . 'px', self::$stylesheet );
 			if ( '' !== self::$script ) {
-				self::$script = 'document.addEventListener("DOMContentLoaded", function(){( function( $ ) { ' . self::$script . ' })(jQuery)})';
+				self::$script = 'document.addEventListener("DOMContentLoaded", () => { ' . self::$script . ' })';
```
Comment (RajkiranBagal) [inline classes/class-uagb-helper.php:190]: Only remove the jQuery function here.

File: README.md [hunk 2]
```diff
@@ -184,6 +184,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Resolved Invalid block issue of core widget blocks.
 * Fix: Marketing Button - Fix Icon size setting issue.
 * Fix: Post Masonary - Fixed the JS UAGBPostMasonry is not defined error.
+* Fix: Social Share - Fixed jquery undefind issue.
```
Comment (RajkiranBagal) [inline README.md:187]: Always check the changelog once before updating. - `jquery => jQuery` & `undefind` => `undefined`.

Update changelog to:
`Fix: Social Share - Fixed the jQuery undefined error.`

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: classes/class-uagb-helper.php (hunk 1 above)
Comment (AbhijitNage123) [reply classes/class-uagb-helper.php]: Updated

File: README.md (hunk 2 above)
Comment (AbhijitNage123) [reply README.md]: Done

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**